static/repos.json

```
@@ -1821,5 +1821,35 @@
                 "1": "1.14+"
             }
         }
+    },
+    "SlimefunGuguProject/SlimeHUD:master": {
+        "type": "maven",
+        "options": {
+            "customDir": "ybw0014/SlimeHUD/master",
+            "target": {
+                "name": "SlimeHUD",
+                "version": "Build {version} zh-CN(ybw0014) (git {git_commit})"
```

署名应为SlimefunGuguProject
```suggestion
                "version": "Build {version} zh-CN(SlimefunGuguProject) (git {git_commit})"
```